elixir/bank-account/lib/account.exs

```diff
@@ -0,0 +1,50 @@
+defmodule Account do
+  use GenServer
+
+  #Client
+  def open_bank do
+    with {:ok, pid} <- GenServer.start_link(__MODULE__, %{balance: 0, status: :open}) do
+      pid
+    else
+      error -> raise "Account was not opened. #{inspect(error)}"
+    end
+  end
+
+  def update(pid, amount) do
+    GenServer.cast(pid, {:deposit, amount})
+  end
+
+  def view(pid) do
+    GenServer.call(pid, :view)
+  end
+
+  def close_bank(pid) do
+    GenServer.cast(pid, :account_closed)
```

Comment:
Maybe you would like to write this as a command rather than past tense:

```suggestion
    GenServer.cast(pid, :close_account)
```

Reply:
Yes, did exactly that in the bank_account file. Sorry about the confusion. I thought I had the accounts file already deleted.